### ISSUE TITLE
Multiline human-friendly descriptions

### ISSUE THREAD

**Issue by ivysotskyi** [lnoor/sphinx-jsonschema#30]
When maintaining JSON files, it's rather difficult to write long multi-line strings, for example in the description field. Sometimes I need to introduce table in documentation and it's nightmare to change such a code:

```
{
"description": "+------------+------------+-----------+ \n| Header 1   | Header 2   | Header 3  | \n+============+============+===========+ \n| body row 1 | column 2   | column 3  | \n+------------+------------+-----------+ \n| body row 2 | Cells may span columns.| \n+------------+------------+-----------+ \n| body row 3 | Cells may  | - Cells   | \n+------------+ span rows. | - contain | \n| body row 4 |            | - blocks. | \n+------------+------------+-----------+"
}
```
Make it possible to write description in more human-friendly way:
```
{
"description": [
  "+------------+------------+-----------+",
  "| Header 1   | Header 2   | Header 3  |",
  "+============+============+===========+",
  "| body row 1 | column 2   | column 3  |",
  "+------------+------------+-----------+",
  "| body row 2 | Cells may span columns.|",
  "+------------+------------+-----------+",
  "| body row 3 | Cells may  | - Cells   |",
  "+------------+ span rows. | - contain |",
  "| body row 4 |            | - blocks. |",
  "+------------+------------+-----------+"
]}
```

**Comment by lnoor** [comment]
I see the value of your suggestion.
I like your solution of using an array for this purpose.

It seems a reasonably minor change however the real problem is that the [standard](http://json-schema.org/latest/json-schema-validation.html#rfc.section.10.1) explicitly states that the value of `description` **MUST** be a string.
And I don't want to violate the standard.

What IS possible though is to extend json-schema with additional keys.
So in line with `$$target` how do you feel about adding `$$description` which allows either strings or arrays of strings?
As an added benefit in your schema you can use them together; where `description` provides a summary and `$$description` more detail.

(note to self: probably need `"\n".join(desc_array)` for this to work)

In the mean time do know that sphinx-jsonschema can read yaml files.
If you're lucky enough that the other software processing your schemas is also able to handle yaml you could [use yaml to have multi line strings](https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-over-multiple-lines)

**Comment by lnoor** [comment]
Hi @ivysotskyi, you're in luck: it is my day off ;-)
I've checked in the change. Looks great, both in source and html.
Since I'm currently not using json schema, could you see if it breaks on a larger project?
Just replace your copy of wide_format.py by the github version.
If it all works I create a new release.

**Comment by ivysotskyi** [comment]
Hi @lnoor ,

I've just tried two big descriptions with big tables and it works perfectly. I will try it for other ones. And in case of any issues let you know. I'm looking forward new release.

**Comment by lnoor** [comment]
Great thanks for the verification.
I've just released version 1.10 on pypi.
So it's official now :)
